data/xml/2022.jlcl.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2022.jlcl">
+  <volume id="1" ingest-date="2025-03-07" type="proceedings">
```

Comment:
Just a note that we should change this to `type="journal"` (for all of the volumes here) and currently have to do this manually, because the ingestion script has no support for that.